<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoliceController</class>
 <widget class="QWidget" name="PoliceController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Police Dashboard</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #E6F0FA; /* Light blue background */
QLabel { font-size: 14px; color: #003087; font-weight: bold; }
QPushButton { background-color: #003087; color: white; border-radius: 5px; padding: 5px; }
QPushButton:hover { background-color: #0052cc; }
QLineEdit { border: 1px solid #003087; border-radius: 3px; padding: 3px; }
QComboBox { border: 1px solid #003087; border-radius: 3px; padding: 3px; background-color: white; }
QTableWidget { border: 1px solid #003087; background-color: white; }
QTableWidget::item { padding: 5px; }
QTableWidget QTableCornerButton::section { background-color: #003087; }
QTableWidget QHeaderView::section { background-color: #003087; color: white; padding: 5px; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <item>
      <widget class="QLabel" name="searchLabel">
       <property name="text">
        <string>Search By:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="searchComboBox">
       <item>
        <property name="text">
         <string>Report ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Driver ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bajaj ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Violation Type</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="placeholderText">
        <string>Enter search term</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sortLayout">
     <item>
      <widget class="QLabel" name="sortLabel">
       <property name="text">
        <string>Sort By:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortComboBox">
       <item>
        <property name="text">
         <string>Report ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Driver ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bajaj ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Violation Type</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sortButton">
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="violationTable">
     <property name="columnCount">
      <number>5</number>
     </property>
     <column>
      <property name="text">
       <string>Report ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Driver ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bajaj ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Violation Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="submitLabel">
     <property name="text">
      <string>Submit New Violation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="driverIdLineEdit">
     <property name="placeholderText">
      <string>Driver ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="bajajIdLineEdit">
     <property name="placeholderText">
      <string>Bajaj ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="violationTypeLineEdit">
     <property name="placeholderText">
      <string>Violation Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionTextEdit">
     <property name="placeholderText">
      <string>Violation Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submitViolationButton">
     <property name="text">
      <string>Submit Violation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
